packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -94,7 +97,14 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
 
   const options = typeof props.options === 'boolean' ? {} : props.options;
 
+  const callStatus = useSelector(getCallStatus);
+  const isLocalMicrophoneEnabled = useSelector(getLocalMicrophoneEnabled);
```

Comment:
Moving this comment to here so it can be an inline conversation without having to use that annoying quote-reply:
> @prprabhu-ms Prathmesh Prabhu FTE @JamesBurnside James Burnside FTE
Couldn't find a clean way to abstract the logic away inside a component level selector in packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

> Any suggestions how it can be done?

> cc: @PorterNan Nan Jiang FTE

Looking it seems the only way is to create a new microphoneButtonSelector and no longer use usePropsFor(MicrophoneButton)? Can usePropsFor work with the selectors defined in the callComposite?

Reply:
I couldn't see a clear way to make usePropsFor work with selectors in callComposite and handlers in component bindings...

Reply:
Disabling microphone toggle as discussed in our standup